package.json

```diff
@@ -16,12 +16,22 @@
     "version-packages": "changeset version",
     "release": "turbo run build --filter=login^... && changeset publish"
   },
+  "pnpm": {
+    "overrides": {
+      "@typescript-eslint/parser": "^7.9.0"
```

Comment:
- https://stackoverflow.com/questions/76996326/parsing-error-deprecationerror-originalkeywordkind-has-been-deprecated-since
- https://github.com/typescript-eslint/typescript-eslint/issues/7155#issuecomment-1692384149